tests/baselines/reference/skipJSDocParsing/see.diff

```diff
@@ -0,0 +1,12 @@
+===================================================================
+--- withJSDoc	With JSDoc
++++ withoutJSDoc	Without JSDoc
+@@ -307,6 +307,6 @@
+     "typeReferenceDirectives": [],
+     "libReferenceDirectives": [],
+     "amdDependencies": [],
+     "identifiers": {},
+-    "jsdocSkipped": false
++    "jsdocSkipped": true
```

Comment:
This is silly but is needed for incremental mode, technically. But I don't think it's possible to actually have this flag set.